index.js

```diff
@@ -62,25 +61,6 @@ module.exports = {
     });
   },
 
-  treeForApp: function (appTree) {
-    var trees = [appTree];
-
-    if (this.includeFactoryGuyFiles) {
-      try {
-        if (fs.statSync('tests/factories').isDirectory()) {
-          var factoriesTree = new Funnel('tests/factories', {
-            destDir: 'tests/factories',
-          });
-          trees.push(factoriesTree);
-        }
-      } catch (err) {
-        // do nothing;
-      }
-    }
-
-    return MergeTrees(trees);
```

Comment:
I stepped through this on the main branch, funnily enough the Funnel throws so the catch block is always hit - this function doesn't do anything, deleted